src/main/resources/fabric.mod.json

```diff
@@ -2,7 +2,7 @@
   "schemaVersion": 1,
   "id": "modmenu",
   "name": "Mod Menu",
-  "version": "$version",
+  "version": "1.8.9",
```

Comment:
Why??

Reply:
This is the mod version. Not the Minecraft version. It needs to be ${version} so it'll reflect whatever the version is set in gradle